jmclient/jmclient/yieldgenerator.py

```diff
@@ -172,9 +177,14 @@ def oid_to_order(self, offer, amount):
         if not filtered_mix_balance:
             return None, None, None
         jlog.debug('mix depths that have enough = ' + str(filtered_mix_balance))
-        mixdepth = self.select_input_mixdepth(filtered_mix_balance, offer, amount)
-        if mixdepth is None:
+
+        try:
+            mixdepth, utxos = self._get_order_inputs(
+                filtered_mix_balance, offer, required_amount)
+        except NoIoauthInputException:
+            jlog.error('unable to fill order, no suitable IOAUTH UTXO found. In order to spend coins (UTXOs) from a mixdepth using coinjoin, there needs to be at least one standard wallet UTXO (not fidelity bond or different address type).')
```

Comment:
any particular reason not to wrap the line?

Reply:
fixed in 8c3ae11